pkgs/development/libraries/ncnn/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchFromGitHub,
+  cmake,
+  opencv, protobuf }:
+
+stdenv.mkDerivation rec {
+  pname = "ncnn";
+  version = "20211208";
+
+  src = fetchFromGitHub {
+    owner = "Tencent";
+    repo = pname;
+    rev = version;
+    sha256 = "1c9axrnafksnks7v5fmi6nzs0qim9n6j5kh5d0vfl3b4r22irhqr";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ opencv protobuf ];
+
+  meta = with lib; {
+    description = "ncnn is a high-performance neural network inference framework optimized for the mobile platform";
+    homepage = "https://github.com/Tencent/ncnn";
+    license = licenses.bsd3;
+    maintainers = with maintainers; [ tilcreator ];
```

Comment:
The maintainer entry doesn't exist yet. You will need to create it in `/maintainers/maintainer-list.nix`.

Reply:
Ups, hoping that #147367 will get merged soon, but should change that here, too ^^

Reply:
#147367 got merged, so this is resolved